config/initializers/slimmer.rb

```diff
@@ -1,7 +1,3 @@
 SmartAnswers::Application.configure do
   config.slimmer.logger = Rails.logger
-
-  if Rails.env.development?
-    config.slimmer.asset_host = ENV["STATIC_DEV"] || Plek.new.find("static")
-  end
```

Comment:
Would this impact the use of `./startup.sh --live`? Which sets:
```
PLEK_SERVICE_STATIC_URI=${PLEK_SERVICE_STATIC_URI-assets.publishing.service.gov.uk}
```
or do we just need to change that to something like?
```
 PLEK_SERVICE_STATIC_URI=gov.uk \
```

Reply:
Thanks for taking a look 👍 

No, that doesn't need to be changed. A confusing quirk of the assets domain is it hosts 3 things:

- Rails static assets for each frontend app
- asset-manager app
- static app

This work only removes the first case of these and the weirdness that static (the app) is hosted on that domain still remains - [some more info here](https://github.com/alphagov/govuk-puppet/pull/10360). Really those files should be served off a static.publishing.service.gov.uk host to clean up some confusion, but hopefully it won't be too long until we kill static.

Reply:
Oh right - make sense! Thanks for the explanation!